devices/bmxx80/bmp180.go

```diff
@@ -0,0 +1,115 @@
+// Copyright 2017 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+package bmxx80
+
+import (
+	"encoding/binary"
+	"time"
+
+	"periph.io/x/periph/devices"
+)
+
+// sense180 reads the device's registers for bmp180.
+//
+// It must be called with d.mu lock held.
```

Comment:
why doesn't this function hold on the `d.mu` if it must be used?

Reply:
Because `Sense()` checks for d.stop as a transaction, and sensing is part of the transaction.
https://github.com/google/periph/pull/156/files#diff-25acf98cebd7f455e61dc13223a25692R155